jobs/candlepinQpidTestsJob.groovy

```diff
@@ -0,0 +1,35 @@
+import jobLib.rhsmLib
+
+String baseFolder = rhsmLib.candlepinJobFolder
+
+String githubOrg = binding.variables['CANDLEPIN_JENKINS_GITHUB_ORG'] ?: 'candlepin'
+
+def rhsmJob = job("$baseFolder/candlepin-pullrequest-spectests-qpid") {
+    previousNames('candlepin-pullrequest-spectests-qpid')
+    environmentVariables(CANDLEPIN_DATABASE: 'postgresql', CP_TEST_ARGS: '-q -r qpid_spec')
+    description("Compiles and deploys Candlepin and runs rspec tests against PostgreSQL with Qpid. Does not produce any installable products.")
+    label('docker')
+    concurrentBuild()
+    wrappers {
+        timeout {
+            noActivity(3600)
+        }
+        preBuildCleanup()
+        colorizeOutput()
+    }
+    logRotator{
+        daysToKeep(720)
+        artifactNumToKeep(5)
```

Comment:
Two years is still a bit much, and I'm not fully understanding the artifactNumToKeep line. Is this keeping artifacts for 5 days, keeping five artifacts, or is it only specifying logs for artifacts?